### PR TITLE
check phase22 sign convention

### PR DESCRIPTION
**Pull request by md-arif-shaikh**
Phase22 should be monotonically increasing. Check whether the phase22 at the merger is greater than that at the start or not. #211

### CODE REVIEW

File: gw_eccentricity/eccDefinition.py [hunk 1]
```diff
@@ -584,6 +584,17 @@ def process_data_dict(self,
         amp22_merger = newDataDict["amplm"][(2, 2)][merger_idx]
         phase22 = newDataDict["phaselm"][(2, 2)]
         phase22_merger = phase22[merger_idx]
+        # check if phase22 is increasing. phase22 at merger should be greater
+        # than the phase22 at the start of the waveform
+        if phase22_merger < phase22[0]:
+            raise Exception(
+                f"Phase22 = {phase22_merger} at the merger is < "
```
Comment (vijayvarma392) [inline gw_eccentricity/eccDefinition.py:591]: Phase22 --> phase22, to be consistent with our own documentation:
https://github.com/vijayvarma392/gw_eccentricity/blob/b96e69dd373dca54fe18ed71676aabda863ff753/gw_eccentricity/eccDefinition.py#L38

**Comment by vijayvarma392** [comment]
@md-arif-shaikh, we should have a similar check that omega22 is positive. Checking that at tmerger is reasonable.

File: gw_eccentricity/eccDefinition.py (hunk 1 above)
Comment (md-arif-shaikh) [reply gw_eccentricity/eccDefinition.py]: Done

**Comment by md-arif-shaikh** [comment]
Both are done @vijayvarma392

**Review by vijayvarma392 — APPROVED** [review]

**Pull request merged**